static/app/views/issueDetails/streamline/issueTagsPreview.tsx

```diff
@@ -39,6 +41,8 @@ const BACKEND_TAGS = [
 const MOBILE_TAGS = ['device', 'os', 'release', 'environment', 'transaction'];
 const RTL_TAGS = ['transaction', 'url'];
 
+const HIGHLIGHT_TAGS = ['handled', 'level', 'mobile', 'main_thread', 'url'];
```

Comment:
i think the problem i have is here. handled and level are usually not interesting and are also default highlight tags